lib/web_api.ex

```diff
@@ -26,7 +26,10 @@ defmodule WebApi do
         # Copy the object to itself, which updates the timestamp, so we only expire files
         # that haven't been used in a while
         {:ok, _} =
-          ExAws.S3.put_object_copy(bucket, key, bucket, key, metadata_directive: :REPLACE)
+          ExAws.S3.put_object_copy(bucket, key, bucket, key,
+            metadata_directive: :REPLACE,
+            meta: [{:text, text}]
```

Comment:
What was the reason we're replacing the metadata with this copy? Is that needed to update the timestamp? It doesn't appear to be part of the metadata — in fact, some quick searching suggests it's almost impossible to _not_ update the timestamp when writing an object, regardless of the method.

Reply:
I think that without directive, the call would error out because it was a no-op. Telling it to replace the metadata makes it an actual change, so it completes and updates the timestamp, which is what we actually care about.